common/src/indexer_service/http/request_handler.rs

```diff
@@ -155,10 +155,10 @@ where
         .map_err(IndexerServiceError::ReceiptError)?;
 
     // Check if we have an attestation signer for the allocation the receipt was created for
-    let signers = state
-        .attestation_signers
-        .value_immediate()
-        .ok_or_else(|| IndexerServiceError::ServiceNotReady)?;
```

Comment:
Have removed the  error check here, since the default value is `HashMap::new()`, hope that's ok?

Reply:
Maybe what we could do to fix this is: Run the function and generate the attestation signers before creating the watcher.

Than you create the watcher that updates, this way we have an "Eventual that is already initialized".

Reply:
Yes, you'd need to refactor not to pass the `&mut Sender`, and just return the `HashMap`.

You could run once before creating and then running inside the tokio task loop.

Reply:
that'd be cool✌️, thanks